release-1.8/release_notes_draft.md

```diff
@@ -37,21 +37,28 @@ please check out the [release notes guidance][] issue.
 
 ## **Major Themes**
 
-- The kubernetes workloads API (the DaemonSet, Deployment, ReplicaSet, and
-StatefulSet kinds) have been moved to the new apps/v1beta2 group version. This
-is the current version of the API, and the version we intend to promote to 
-GA in future releases. This version of the API introduces several deprecations 
-and behavioral changes, but its intention is to provide a stable, consistent 
-API surface for promotion.
-
-- The roles based access control (RBAC) API group for managing API authorization
-has been promoted to v1. No changes were made to the API from v1beta1. This
-promotion indicates RBAC's production readiness and adoption. Today, the
-authorizer is turned on by default by many distributions of Kubernetes, and is a
-fundamental aspect of a secure cluster.
+### Versioning changes
+
+Version changes were made to the following APIs. For details, see 
+<!-- need to figure out correct links. "Action Required Before Upgrading", 
+but also other sections with more detail. The goal is to reduce duplication/
+redundancy -->.
+
+- Workloads (`DaemonSet`, `Deployment`, `ReplicaSet`, `StatefulSet`)
+
+- RBAC
+
+- Autoscaling
+
+- Metrics
+
+- Advanced auditing
 
 ## **Action Required Before Upgrading**
```

Comment:
Can we simplify this title to "Prerequisites for Upgrading to v1.8"

Reply:
Well, it isn't really prerequisites -- it's things you need to know before you upgrade. Not quite the same thing ... rn anyway all three items describe changes you need to make to your config, part of one item is about upgrading a component (should be its own item too), and part of another item is about output changes that affect (I assume) how you parse the output.
That said, I agree that we could still come up with a better section title. I'll ponder some more.

Reply:
Yes, the changes we should make before upgrading, which is nothing but prerequisites (requirements) right?